Wrappers/Python/test/test_functions.py

```diff
@@ -940,6 +943,43 @@ def proximal_conjugate_test(self, function, geom):
             np.testing.assert_allclose(a.as_array(), b.as_array(),
                                     rtol=1e-5, atol=1e-5)
 
+    def test_L1Norm_vs_WeightedL1Norm_noweight(self):
+        f1 = L1Norm()
+        f2 = WeightedL1Norm(weight=None)
+
+        assert f1.__class__.__name__ == 'L1Norm'
+        assert f2.__class__.__name__ == 'L1Norm'
+
+    def test_L1Norm_vs_WeightedL1Norm(self):    
```

Comment:
We should be able to test a case when we don't just allocate 1s to the weights. 

e.g. weights= geom.allocate('random')
x=geom.allocate(1)
f1=L1Norm()
f2=weightedL1Norm(weights=weights)
np.testing.assertEqual(f2(x), f1(x))